### PR TITLE
added division

### PR DESCRIPTION
**Pull request by KapitanMarbel**
naglagay ako ng division hehe

### CODE REVIEW

File: main1.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	int a, b, c;
+	cout<< "Please enter a number: ";
+	cin>> a;
+	
+	cout<< "Please enter a number: ";
```
Comment (Nelliosis) [inline main1.cpp]: I want cin a and b in the same line

Comment (KapitanMarbel) [reply main1.cpp]: ayt

**Comment by Nelliosis** [comment]
Goodu jobbu

**Comment by Nelliosis** [comment]
Goodu jobbu

**Pull request merged**